pkg/instance/instance.go

```diff
@@ -59,3 +70,29 @@ func (i *Info) UpgradeRequired() bool {
 	// fully deconfigured before being configured by the current version.
 	return true
 }
+
+// getIPv4Address returns an IPv4 address associated with the given address. An error will be thrown if given
+// an IPv6 address or a DNS address that does not resolve to an IPv4 network address.
+func getIPv4Address(address string) (string, error) {
```

Comment:
nit: the naming makes it sound like a getter
consider something else like `resolveIPv4Address`
not a big deal considering this isnt a method of the `Info` struct.

Reply:
Agreed, will make a change to `resolveToIPv4Address`